### PR TITLE
Doc: Update TestNet documentation about consensus parameters

### PR DESCRIPTION
**Pull request by mkykadir**
Consensus parameters are needed to sync with network and `--testnet` flag
does this by default. Documentation was misleading when providing own
configuration.

### CODE REVIEW

File: devel/testnet/README.md [hunk 1]
```diff
@@ -71,6 +71,8 @@ Agora can be extensively configured via the use of a configuration file.
 By default, Agora will look for `config.yaml` in the directory it is started in, but using the `-c`
 command line option, you can give Agora any configuration file.
 Using `--testnet` is a special way to make Agora start without a configuration file.
+`--testnet` flag is still required to fetch `consensus` parameters from test network,
+even a configuration file is provided.
```
Comment (Geod24) [inline devel/testnet/README.md]: That should be "even **if**". But I'd rephrase it as just saying something along the line of"Always use `--testnet`"

Comment (mkykadir) [reply devel/testnet/README.md]: Updated

**Review by Geod24 — APPROVED** [review]
